kubernetes/base/kustomization.yaml

```diff
@@ -8,6 +8,8 @@ resources:
 - service-account.yaml
 - service-external.yaml
 - service-internal-discovery.yaml
+- chat-alertrule.yaml
+- configmap.yaml
```

Comment:
Is there any reason not to create this configmap with the format used below ?
```
- name: ejabberd-statsd-mapping
  files:
  - config/statsd-mapping.conf
```
If not, let's follow the same pattern we're using for our other configmaps. If there is let's at least rename this file to statsd-mapping-configmap because it's not the only configmap that we create for ejabberd